### DISCUSSION TITLE
Check existing record with long URL

### DISCUSSION THREAD

**Discussion by nyunity** [YOURLS/YOURLS#3434] [General]
Hello @all!

is it possible to use a action for get some xml information with the long url? 

What I want: Check my long URL against the database, but in XML/URL form, like it work with the short url function/action url-stats.

Thanks,

Erik

**Comment by dgw** [comment]
This is something that would be possible with a plugin.

You can compose this without writing any code using https://github.com/brookestevens/yourls-api-contract (or https://github.com/bryzgaloff/yourls-api-lookup-keywords-by-url-substr) to get the short URL(s) that match a long URL, and then look up the stats in a second API request.

**Reply by nyunity** [reply]
works great, thank you!

**Comment by nyunity** [comment]
again, sorry for next reply, but I am not able to solve this:

I look for a long url with parms like https://googl.de?parm1=123&parm2=this+is+a+var and I do not get the right url, because it stops before "&". It does not work with urlencode too... 

Do you know a solution?

thx, 
Erik
